docs/adrs/0074-move-history-details-refactor.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "0074 Refactor Move Event's Details Rendering "
+---
+
+# *0074 Refactor Move Event's Details rendering*
+
+**🔒 User Story:** *[MB-11214](https://dp3.atlassian.net/browse/MB-12606)*
+
+*At present, move history templates are matched based on action, event name, and table name. This expects that each action, event, and table combination is unique. However, some move history events share the same values for all three and there isn't a way to specify which template to use.*
+
+### Useful definition
+> **Matching combination:** A move event's action, eventName, and tableName.
+
+## Example of two move events with the same action, event, and table combination
+
+<table>
+  <tr>
+    <td>Approvals request</td>
+    <td>Prime excess weight </td> 
+  </tr>
+  <tr>
+    <td>
+
+```json
+{
+  "action": "UPDATE",
+  ...
+  "changedValues": {
+    "status": "APPROVALS REQUESTED"
+  },
+  ...
+  "eventName": "updateMTOShipment",
+  "tableName": "moves",
+  "transactionId": 6627,
+}
+```
+
+  </td>
+    <td>
+
+```json
+{
+  "action": "UPDATE",
+  ...
+  "changedValues": {
+    "excess_weight_qualified_at": "2022-09-07T21:26:47"
+  },
+  ...
+  "eventName": "updateMTOShipment",
+  "tableName": "moves", 
+  "transactionId": 6627,
+}
+```
+
+  </td>
+  </tr>
+</table>
+
+> **Note:**
+> History record data has between abbreviated.
+
+
+**Above there are two issues:**
+1. The move events will match multiple templates
```

Comment:
**suggestion**(non-blocking): you can probably remove these two lines since the following descriptions of the issues already have their own titles